### ISSUE TITLE
Cannot find module 'nativewind/metro' 

### ISSUE THREAD

**Issue by Savinvadim1312** [danstepanov/create-expo-stack#318]
## Expected Behavior
The metro config file should not import `const { withNativeWind } = require('nativewind/metro');` when "StyleSheet" styling is selected.

## Steps to Reproduce
1. Init a new project with `npx create-expo-stack@latest` Version 2.10.2
2. Select Stylesheet for styling
3. Run the project with `npm start`
4. The error is thrown: `Error: Cannot find module 'nativewind/metro'`

Temporary workaround: remove the `const { withNativeWind } = require('nativewind/metro');` from `metro.config.js`

## Context (Environment)
Project configuration:
```
{
  projectName: 'my-expo-app',
  packages: [
    {
      name: 'expo-router',
      type: 'navigation',
      options: { type: 'stack' }
    },
    { name: 'stylesheet', type: 'styling' },
    { name: 'supabase', type: 'authentication' }
  ],
  flags: {
    noGit: false,
    noInstall: false,
    overwrite: false,
    importAlias: true,
    packageManager: 'npm',
    eas: false
  }
}
```

**Comment by Savinvadim1312** [comment]
In this file: https://github.com/danstepanov/create-expo-stack/blob/main/cli/src/templates/packages/expo-router/metro.config.js.ejs

I found:

```
<% if (props.stylingPackage?.name === "nativewind" || "nativewindui") { %>
  const { withNativeWind } = require("nativewind/metro");
<% } %>
```

Is it supposed to be:
```
<% if (props.stylingPackage?.name === "nativewind" || props.stylingPackage?.name === "nativewindui") { %>
  const { withNativeWind } = require("nativewind/metro");
<% } %>
```

I haven't tested it, but seems like the problem comes from here

**Comment by danstepanov** [comment]
Thanks for flagging this! I'll resolve it today 🙂

**Comment by danstepanov** [comment]
Resolved via #319
